modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -10,6 +10,12 @@ The `install-config.yaml` file requires some additional details.
 Most of the information is teaching the installer and the resulting cluster enough about the available hardware so that it is able to fully manage it.
 
 . Configure `install-config.yaml`. Change the appropriate variables to match the environment, including `pullSecret` and `sshKey`.
+
+[NOTE]
+====
+In {product-title} 4.4 and later releases, the `bootMacAddress` field is used to calculate the IP address for Ironic to bind to. The `provisioningNetworkInterface` configuration field is optional.
```

Comment:
Maybe:
...configuration field is now optional.

Reply:
Updated as suggested.